qiskit/pulse/schedule/pulse_schedule.py

```diff
@@ -70,7 +70,7 @@ class PulseSchedule(TimedPulseBlock):
     """Schedule."""
 
     def __init__(self,
```

Comment:
What do you think about providing the ability to initalize the `PulseSchedule`, with either a `backend+kwargs` or a `channel_store`?. This would remove one (what might be seen as extraneous) step from the `PulseSchedule` creation with the (small?) downside of creating multiple ways to initialize.

Reply:
I disagree with including `backend` in the arguments of `PulseSchedule`'s constructor. I think we should not make `PulseSchedule` depend on `Backend` because executing the same schedule on several backends is a conceivable use case.

Reply:
In the above use case the `schedule` could be initialized with `channel_store`.